app/views/shared/authentication/_email_registration_form.html.erb

```diff
@@ -4,9 +4,9 @@
       <%= flash[:notice] %>
     </div>
   <% end %>
-  <%= form_for(User.new, as: :user, url: registration_path(:user)) do |f| %>
+  <%= form_for(User.new, as: :user, data: { testid: "registration-form" }, url: registration_path(:user)) do |f| %>
```

Comment:
These data-testid attributes are pretty standard these days. See [Making your UI tests resilient to change](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change).